composer.json

```diff
@@ -12,25 +12,31 @@
             "homepage": "http://kriswallsmith.net/"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/WouterJ/assetic"
+        }
+    ],
```

Comment:
Must be removed once https://github.com/kriswallsmith/assetic/pull/772 is merged.